clang/include/clang/Driver/Options.td

```diff
@@ -7145,10 +7145,13 @@ defm sycl_force_inline_kernel_lambda
           NegFlag<SetFalse, [], [ClangOption, CLOption], "Disallow">,
           BothFlags<[], [ClangOption, CLOption, CC1Option], " force inline "
                     "SYCL kernels lambda in entry point">>;
-def fsycl_help_EQ : Joined<["-"], "fsycl-help=">, Flags<[NoXarchOption]>,
-  HelpText<"Emit help information from the related offline compilation tool. "
-           "Valid values: all, fpga, gen, x86_64.">,
-  Values<"all,fpga,gen,x86_64">;
+def fsycl_help_EQ
```

Comment:
Should this be removed:
https://github.com/intel/llvm/blob/fb70c93770981a9ea1ae74d8541de56a150078c9/clang/include/clang/Driver/Options.td#L7019

Thanks

Reply:
No, not yet.  We keep it in place as the use of `-fintelfpga` emits a special diagnostic upon usage due to the fact that we removed these without much of a deprecation period.  We can do full removal with a major release cycle.